src/test/java/org/folio/linked/data/e2e/AuthorityAssignmentControllerIT.java

```diff
@@ -0,0 +1,62 @@
+package org.folio.linked.data.e2e;
+
+import static java.lang.String.valueOf;
+import static org.folio.linked.data.test.TestUtil.defaultHeaders;
+import static org.folio.linked.data.test.TestUtil.loadResourceAsString;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.folio.linked.data.domain.dto.AssignmentCheckDto;
+import org.folio.linked.data.e2e.base.IntegrationTest;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.test.web.servlet.MockMvc;
+
+@IntegrationTest
+public class AuthorityAssignmentControllerIT {
+
+  public static final String RESOURCE_URL = "/linked-data/authority-assignment-check";
+
+  @Autowired
+  private Environment env;
+
+  @Autowired
+  private MockMvc mockMvc;
+
+  @Autowired
+  private ObjectMapper objectMapper;
+
+  @ParameterizedTest
+  @CsvSource({
+    "samples/marc/authority_person.json, true",
+    "samples/marc/authority_family.json, true",
+    "samples/marc/authority_organization.json, true",
+    "samples/marc/authority_jurisdiction.json, true",
+    "samples/marc/authority_concept_meeting.json, true",
```

Comment:
This should be false. Resource of type "Concept, Meeting" cannot be assigned as the creator of a work.

Reply:
fixed.